packages/schematics/angular/config/index.ts

```diff
@@ -103,3 +105,49 @@ function addKarmaConfig(options: ConfigOptions): Rule {
     );
   });
 }
+
+function addAiContextFile(options: ConfigOptions): Rule {
+  let fileName: string;
+  let filePath: string;
+  switch (options.tool) {
+    case 'gemini':
+      fileName = 'GEMINI.md';
+      filePath = '.gemini';
+      break;
+    case 'copilot':
+      fileName = 'copilot-instructions.md';
+      filePath = '.github';
+      break;
+    case 'claude':
+      fileName = 'CLAUDE.md';
+      filePath = '.claude';
+      break;
+    case 'cursor':
+      fileName = 'cursor.md';
+      filePath = path.join('.cursor', 'rules');
+      break;
+    case 'windsurf':
+      fileName = 'guidelines.md';
+      filePath = path.join('.windsurf', 'rules');
+      break;
+    // TODO: should the default be 'all'?
+  }
+
+  return async (host) => {
+    const workspace = await readWorkspace(host);
+    const project = workspace.projects.get(options.project);
+    if (!project) {
+      throw new SchematicsException(`Project name "${options.project}" doesn't not exist.`);
+    }
+
+    return mergeWith(
+      apply(url('./files'), [
+        // Keep this file in sync with the one presented here https://angular.dev/ai/develop-with-ai
+        // The file is located in the framework repo at adev/src/context/best-practices.md
+        filter((p) => p.endsWith(`__rulesName__.template`)),
+        applyTemplates({ rulesName: fileName, directoryName: filePath }),
```

Comment:
Question: Do we need different frontmatter for each of these files, just to apply by default or other configuration to set?

I was expecting these files to be slightly different while containing the same core content.